server/sonar-server/src/main/java/org/sonar/server/measure/ws/TimeMachineWs.java

```diff
@@ -42,7 +42,7 @@ private void defineSystemAction(NewController controller) {
     NewAction action = controller.createAction("index")
       .setDescription("Get a list of past measures. Requires Browse permission on project")
       .setSince("2.10")
-      .setHandler(RailsHandler.INSTANCE)
+      .setHandler(RemovedWebServiceHandler.INSTANCE)
```

Comment:
I also need to rename the commit message as my first objective was to remove this commit